Makefile

```diff
@@ -315,7 +315,7 @@ ARKOUDA_SOURCES = $(shell find $(ARKOUDA_SOURCE_DIR)/ -type f -name '*.chpl')
 ARKOUDA_MAIN_SOURCE := $(ARKOUDA_SOURCE_DIR)/$(ARKOUDA_MAIN_MODULE).chpl
 
 ifeq ($(shell expr $(CHPL_MINOR) \>= 30),1)
-	CHPL_FLAGS += -sbigintInitThrows=true
+	CHPL_COMPAT_FLAGS += -sbigintInitThrows=true
```

Comment:
Could/should this use `OPTIONAL_SERVER_FLAGS`?

Reply:
So you are saying, rather than `CHPL_COMPAT_FLAGS`, we'd use `OPTIONAL_SERVER_FLAGS`? 

I don't know if I would go that route, since it isn't really an "optional" flag (you need it with 1.30). Though, I guess the definition of "need" here is debatable, since it is just quieting a deprecation warning.